cypress/integration/functional/onboarding/createMnemonicWallet.spec.ts

```diff
@@ -68,3 +68,46 @@ context('Onboarding - Create Mnemonic Wallet', () => {
     })
   })
 })
+
+context('Onboarding - Create Mnemonic Wallet with refresh recovery word', () => {
+  const recoveryWords: string[] = []
+  const oldEecoveryWords: string[] = []
+
+  beforeEach(() => {
+    cy.restoreLocalStorage()
+    cy.visit('/')
+    cy.exitWallet()
+  })
+
+  afterEach(() => {
+    cy.saveLocalStorage()
+  })
+
+  it('should refresh recovery word', function () {
```

Comment:
Add a check here that the refreshed recovery words can be restored after creation.